docs/architecture/systems/agentkit-forge.md

```
@@ -0,0 +1,192 @@
+# AgentKit Forge
+
+AgentKit Forge builds AI agents and orchestration workflows. SLMs help when agents have **many tools** and **large working memory**.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)